src/components/structures/MyGroups.js

```diff
@@ -19,23 +20,23 @@ import PropTypes from 'prop-types';
 import sdk from '../../index';
 import { _t } from '../../languageHandler';
 import dis from '../../dispatcher';
-import withMatrixClient from '../../wrappers/withMatrixClient';
 import AccessibleButton from '../views/elements/AccessibleButton';
+import * as Matrix from "matrix-js-sdk";
```

Comment:
It's better to import only what you need rather than all of the SDK like this, as it's clearer what's being used, and it's easier for tools to do more thorough tree shaking in the future.

Something like:

```suggestion
import { MatrixClient } from "matrix-js-sdk";
```

here and in other files.

Reply:
I copied this from https://github.com/matrix-org/matrix-react-sdk/blob/f80cbebb9dd79a3aa0e9c15e99fff9b6d51b73ad/src/wrappers/withMatrixClient.js#L18

Reply:
done anyway